nemo/utils/arguments.py

```diff
@@ -103,3 +103,28 @@ def add_asr_args(parent_parser: ArgumentParser) -> ArgumentParser:
     parser.add_argument("--train_dataset", type=str, required=True, default=None, help="training dataset path")
     parser.add_argument("--eval_dataset", type=str, required=True, help="evaluation dataset path")
     return parser
+
+
+def add_nlp_args(parent_parser: ArgumentParser) -> ArgumentParser:
+    """Extends existing argparse with default NLP collection args.
+
+    Args:
+        parent_parser (ArgumentParser): Custom CLI parser that will be extended.
+
+    Returns:
+        ArgumentParser: Parser extended by NeMo NLP Collection arguments.
+    """
+    parser = ArgumentParser(parents=[parent_parser], add_help=False, conflict_handler='resolve')
+    parser.add_argument(
+        "--data_dir", type=str, required=True, help="data directory to training or/and evaluation dataset"
+    )
+    parser.add_argument("--config_file", type=str, required=False, help="model configuration file")
```

Comment:
should we explicitly say in help that this is HF config file? Or is it?

Reply:
it is will add